openquake/qa_tests_data/classical/case_1/expected/hazard_curve-PGA.csv

```diff
@@ -1,3 +1,3 @@
-# generated_by='OpenQuake engine 3.5.0-gitd83c42cda6', start_date='2019-05-12T07:12:05', checksum=141718627, kind='rlz-000', investigation_time=1.0, imt='PGA'
+#,,,,,"generated_by='OpenQuake engine 3.7.0-git5e91b20b88', start_date='2019-09-05T06:13:25', checksum=141718627, kind='rlz-000', investigation_time=1.0, imt='PGA'"
```

Comment:
Why all the commas at the beginning?

Reply:
I originally put the commas at the beginning but then it looks bad on GitHub :-(